### PR TITLE
keadm: Installer may be disabled if offline mode is desired

### PR DESCRIPTION
**Pull request by rzr**
Configurations steps are still here.

For smooth transition, the curent behavior is preserved.

Change-Id: Icbf96b5e2b2179671bb09d8f7b5152c547038b70
Relate-to: https://github.com/kubeedge/kubeedge/issues/1117
Signed-off-by: Philippe Coval <p.coval@samsung.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: keadm/app/cmd/util/common.go [hunk 1]
```diff
@@ -32,6 +32,8 @@ import (
 
 //Constants used by installers
 const (
+	InstallersOnline = true
```
Comment (fisherxu) [inline keadm/app/cmd/util/common.go]: How to change the mode mow? :)

File: keadm/app/cmd/util/edgecoreinstaller.go [hunk 2]
```diff
@@ -51,9 +51,12 @@ func (ku *KubeEdgeInstTool) InstallTools() error {
...
 
-	err := ku.InstallKubeEdge()
-	if err != nil {
-		return err
+	var err error
+	if InstallersOnline {
+		err := ku.InstallKubeEdge()
+		if err != nil {
+			return err
+		}
```
Comment (fisherxu) [inline keadm/app/cmd/util/edgecoreinstaller.go]: In cloudcore, If offline mode, we only start binary, don't copy again, why not add https://github.com/kubeedge/kubeedge/pull/1266/files#diff-f7a2c5d5ad972d622b21fe88054b53c3R184-R193 here?

File: keadm/app/cmd/util/common.go (hunk 1 above)
Comment (rzr) [reply keadm/app/cmd/util/common.go]: patch source ? this is what i am doing with by my debian packaging branch maybe I could add an --offline flag ?

File: keadm/app/cmd/util/edgecoreinstaller.go (hunk 2 above)
Comment (rzr) [reply keadm/app/cmd/util/edgecoreinstaller.go]: thx maybe I'll share an other patch

Is the feature still desirable ?

/assign @kevin-wangzefeng

**Pull request closed without merging**